### PR TITLE
Added Campagin post export functionality.

### PR DESCRIPTION
**Pull request by onyekaa**
This is a repost of @mehulkaklotar's previous branch/PR: #683 . I rebased from the updated branch `planet-2561`. See below:

====

Ref: https://jira.greenpeace.org/browse/PLANET-2656

Import & Export Campaigns
Note: Wordpress Importer plugin is used for importing the xml file

### CODE REVIEW

File: classes/class-p4-campaign-exporter.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	}
+
+	/*
+	 * Add Export Link
+	 */
+	public function single_post_export( $actions, $post ) {
+		if ( ! empty( $_GET['post_type'] ) &&
+		     'campaigns' == $_GET['post_type'] &&
+             current_user_can( 'edit_posts' ) ) {
+			$actions['export'] = '<a href="admin.php?action=export_data&amp;post=' . $post->ID . '" title="' . __( 'Export', 'planet4-master-theme-backend' ) . '" rel="permalink">' . __( 'Export', 'planet4-master-theme-backend' ) . '</a>';
```
Comment (mikelittle) [inline classes/class-p4-campaign-exporter.php]: This url should really be created with `admin_url()` and escaped.

File: exporter.php [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+		echo '<wp:author_login>' . $author->user_login . '</wp:author_login>';
+		echo '<wp:author_email>' . $author->user_email . '</wp:author_email>';
+		echo '<wp:author_display_name>' . p4_px_single_post_cdata( $author->display_name ) . '</wp:author_display_name>';
+		echo '<wp:author_first_name>' . p4_px_single_post_cdata( $author->user_firstname ) . '</wp:author_first_name>';
+		echo '<wp:author_last_name>' . p4_px_single_post_cdata( $author->user_lastname ) . '</wp:author_last_name>';
+		echo "</wp:author>\n";
+	}
+}
+
+function p4_px_single_post_nav_menu_terms() {
```
Comment (mikelittle) [inline exporter.php]: Do we need support for nav menu terms in this export file?

Comment (onyekaa) [reply exporter.php]: Don't believe so. Removed.

**Comment by onyekaa** [comment]
Updated branch, and rebased. 
@mikelittle @koyan

**Pull request merged**